packages/react/package.json

```diff
@@ -18,6 +18,7 @@
     "access": "public"
   },
   "dependencies": {
+    "use-sync-external-store": "^1.2.0",
```

Comment:
Is it compatible with react@16? _if not_, let's upgrade react to @>=17.0.0 in dev/peer dependencies

Reply:
compatible